pkg/controller/volume/persistentvolume/BUILD

```diff
@@ -37,6 +37,7 @@ go_library(
         "//staging/src/k8s.io/api/storage/v1:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/api/errors:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/api/meta:go_default_library",
+        "//staging/src/k8s.io/apimachinery/pkg/api/resource:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/apis/meta/v1:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/labels:go_default_library",
```

Comment:
We are missing a change expand_controller which is https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/expand/expand_controller.go 

We need same change you did https://github.com/kubernetes-csi/external-resizer/pull/140/files in intree controller.